src/admin/content-block/content-block.types.ts

```diff
@@ -1,5 +1,7 @@
 import { ButtonAction, IconName } from '@viaa/avo2-components';
+// TODO: Import from components lib when exported there.
 import { ButtonType } from '@viaa/avo2-components/dist/components/Button/Button.types';
+import { GridItem } from '@viaa/avo2-components/dist/content-blocks/BlockGrid/BlockGrid';
```

Comment:
Ik stel voor dat we bovenstaande exporten in components library want anders vergeten we dit nog.